### DISCUSSION TITLE
240321

### DISCUSSION THREAD

**Discussion by jisunipark** [codeit-part4-8team-project/main#61] [팀회의]
## 멘토님 질문 사항

**Comment by jisunipark** [comment]
## 논의사항
![image](https://github.com/codeit-part4-8team-project/main/assets/148641571/287a6b52-0514-4de6-88c7-11b55def6b27)
모달 열었을 때 포지션 문제

![image](https://github.com/codeit-part4-8team-project/main/assets/148641571/f8c230ad-0e52-4ac2-a43f-a21aab8acee3)
메인페이지 캘린더 width 문제

## 어제 한 일
- 칸반보드 관련 부분 리팩토링
- 데이터 구조 반영

## 오늘 할 일
- 자유게시판 / 칸반보드 ui 및 데이터 구조 반영해서 pr 올리기

## 오늘의 일정 (코어 타임)
- 1:40분 병원 (1:20 - 2:30 자리 비울 예정)

## 멘토님 질문 사항

**Comment by MoonPillGyeom** [comment]
## 논의사항
- x
## 어제 한 일
- 리팩토링
## 오늘 할 일
- 남은 모달 작업
## 오늘의 일정 (코어 타임)
- 열심히 코딩하기
## 멘토님 질문 사항
- x

**Comment by caseV12** [comment]
## 논의사항

## 어제 한 일
소셜 로그인 구현, 회원가입 쪽 api 테스트, 훅 폼 작성, axios 훅 수정, 공통 인풋컴포넌트 작성
## 오늘 할 일
회원가입 쪽 에러 문의드리고 해결해서 회원가입 완성
## 오늘의 일정 (코어 타임)
x
## 멘토님 질문 사항
